handler/core/src/test/java/io/knotx/fragments/handler/action/CircuitBreakerActionTest.java

```diff
@@ -36,11 +47,6 @@
 import io.vertx.junit5.VertxExtension;
 import io.vertx.junit5.VertxTestContext;
 import io.vertx.rxjava.core.Future;
-import java.util.concurrent.TimeUnit;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
 
 @ExtendWith(VertxExtension.class)
 class CircuitBreakerActionTest {
```

Comment:
I think that the CB is very important when we talk about the action's log. This log should contain all elements that allows us to monitor its state.